src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/CollidingNodeSystemAlgorithm/Vertex.java

```diff
@@ -0,0 +1,58 @@
+package edu.greenblitz.tobyDetermined.Nodesssss.CollidingNodeSystemAlgorithm;
+
+import static edu.greenblitz.tobyDetermined.Nodesssss.NodeSystemUtils.*;
+import static edu.greenblitz.tobyDetermined.Nodesssss.NodeBase.SystemsPosition;
+
+
+import java.util.LinkedList;
+
+public class Vertex {
+    private final SystemsPosition startPosition;
+    private final SystemsPosition endPosition;
+    private SystemsPosition system2Position;
```

Comment:
I would suggest to write 'to' instead of 2 for clarity

Reply:
it means system number 2 position not system to position